charts/gardener-extension-shoot-cert-service/templates/poddisruptionbudget.yaml

```
@@ -12,6 +12,4 @@ spec:
     matchLabels:
       app.kubernetes.io/name: gardener-extension-shoot-cert-service
       app.kubernetes.io/instance: {{ .Release.Name }}
-{{- if semverCompare ">= 1.26-0" .Capabilities.KubeVersion.GitVersion }}
-  unhealthyPodEvictionPolicy: AlwaysAllow
-{{- end }}
+  unhealthyPodEvictionPolicy: AlwaysAllow
```

Nit: Let's keep the trailing newline
```suggestion
  unhealthyPodEvictionPolicy: AlwaysAllow

```